src/sth_database.js

```diff
@@ -342,6 +374,42 @@
   }
 
   /**
+   * Saves the contents of a readable stream into a file in the local file system
+   * @param {string} attrName The name of the attribute the stream contents refers to
+   * @param {object} stream The stream
+   * @param {string} fileName The file name where the stream contents should be stored
+   */
+  function save2File(attrName, stream, fileName) {
+    // Create the directory for the logging files if it does not exist
+    if (fs.existsSync(sthConfig.TEMPORAL_DIR)) {
+      // The path exists
+      if (!fs.statSync(sthConfig.TEMPORAL_DIR).isDirectory()) {
+        // The paths exists but it is not a directory
+        mkdirp.sync(sthConfig.TEMPORAL_DIR);
```

Comment:
don't it do the same thing on every branch?  

mkdirp.sync(sthConfig.TEMPORAL_DIR);